README.md

```diff
@@ -119,7 +120,29 @@ that specific token. If omitted, the approval covers all tokens.
 
 The following APIs are not part of the fftokens standard, but are exposed under `/api/v1`:
 
-* `GET /receipt/:id` - Get receipt for a previous request
+- `GET /receipt/:id` - Get receipt for a previous request
+
+## Retry behaviour
```

Comment:
I'd probably move this section further toward the end, as I'd consider it a pretty deep/advanced bit of configuration. Probably less important than "how do I run this" and "how do I run unit tests".

Reply:
Good thought, I've moved it to the bottom of the README